quickannotator/api/v1/tile/helper.py

```diff
@@ -93,9 +94,17 @@ def get_tile_id_for_point(tile_size: int, x: int, y: int, image_width: int, imag
 
     col = x // tile_size
     row = y // tile_size
-    tile_id = row * math.ceil(image_width / tile_size) + col + 1  # Adjust tile_id to start at 1
+    tile_id = row * math.ceil(image_width / tile_size) + col
     return tile_id
 
+def get_tile_id_for_rc_index(tile_size: int, row: int, col: int, image_width: int, image_height: int) -> int:
```

Comment:
note that in one of my DL scripts, i put a comment which says we'll as well need the inverse function. given a tileid convert to row/col